app/renderer/components/styles/commonStyles.js

```diff
@@ -19,6 +19,17 @@ const styles = StyleSheet.create({
     outline: 'none',
     padding: '0.4em',
     width: '100%'
+  },
+  'flyoutDialog': {
+    backgroundColor: `${globalStyles.color.toolbarBackground}`,
+    borderRadius: `${globalStyles.radius.borderRadius}`,
```

Comment:
Do we need the blackets here? I think this should work: `backgroundColor: globalStyles.color.toolbarBackground`. cf: https://github.com/brave/browser-laptop/pull/7107/files#diff-fa59d1e0221a5bee94fd2abd9b96dcfdR12